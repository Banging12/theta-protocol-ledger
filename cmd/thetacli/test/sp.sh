#!/usr/bin/expect -f
set timeout -1
set arg1 [lindex $argv 0]
set arg2 [lindex $argv 1]
set arg3 [lindex $argv 2]
set arg4 [lindex $argv 3]
set arg5 [lindex $argv 4]
set arg6 [lindex $argv 5]
set arg7 [lindex $argv 6]
set arg8 [lindex $argv 7]
spawn thetacli tx service_payment --chain="privatenet" $arg1 $arg2 $arg3 $arg4 $arg5 $arg6 $arg7 $arg8
set code [open "./defaultpw" r]
set pass [read $code]
expect {
        password: {send "$pass\r" ; exp_continue}
        eof exit
}
