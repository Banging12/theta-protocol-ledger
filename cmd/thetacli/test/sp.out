spawn thetacli tx service_payment --chain="privatenet" --from=2E833968E5bB786Ae419c4d13189fB081Cc43bab --to=70f587259738cB626A1720Af7038B8DcDb6a42a0 --payment_seq=1 --reserve_seq=3 --resource_id=rid1000001 --tfuel=2
Using config file: /Users/i830671/.thetacli/config.yaml
Please enter password: qwertyuiop



Off-Chain transaction:
{
    "fee": {
        "thetawei": "0",
        "tfuelwei": "1000000000000"
    },
    "source": {
        "address": "0x2e833968e5bb786ae419c4d13189fb081cc43bab",
        "coins": {
            "thetawei": "0",
            "tfuelwei": "2000000000000000000"
        },
        "sequence": "1",
        "signature": "0x5dc0a9661ad351c3d62e42359e9d514345b40fe93679578efb0140ebd1130d904598892ac5c444e96287855ec2a684cf9ba162b2ffe72feca8540a92f83a9ad500"
    },
    "target": {
        "address": "0x70f587259738cb626a1720af7038b8dcdb6a42a0",
        "coins": {
            "thetawei": "0",
            "tfuelwei": "2000000000000000000"
        },
        "sequence": "0",
        "signature": "0x756e7369676e6564"
    },
    "payment_sequence": "1",
    "reserve_sequence": "3",
    "resource_id": "rid1000001"
}
